src/main/scala/io/github/pauljamescleary/petstore/endpoint/JodaDateTime.scala

```diff
@@ -5,7 +5,8 @@ import org.joda.time.DateTime
 import org.joda.time.format.DateTimeFormat
 
 object JodaDateTime {
-  private val dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS"
+  // you should always add ZZ for the timezone :)
+  private val dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZZ"
```

Comment:
IMO, we should probably use java.time from 1.8, but it's minor.

In my experience, most of the problems with date, time and timezone is because we use them to store an instant in time. time zone depends on the user preference most of the time. What you want in most of the case is something like: the event occurred on 2017-10-31T12:26:26 America/New York and that date time should be translated to a epoch or some format that is timezone agnostic.

Then, when displayed to the user, the user timezone is used to properly format the date time

Reply:
You are right, the problem is what is store in db. We should ALWAYS store it in UTC or with the local of the place where is was generated so that we can latter display the correct date/time to the user (in user local). At least here, the user knows what was the timezone. But I agree, it's minor.

Reply:
Datetimes I always store as UTC in the database, and then it is up to the app to determine how to present them.

I like this update

Reply:
Seems like maybe circe has first class support for Instant, will check it out

Reply:
woot!  Changed JodaDateTime to Instant... :heavy_check_mark: